cookbook/case_studies/ml_training/spark_horovod/Dockerfile

```diff
@@ -115,5 +118,3 @@ ENV FLYTE_INTERNAL_IMAGE $tag
 RUN cp ${VENV}/bin/flytekit_venv /usr/local/bin/
 RUN chmod a+x /usr/local/bin/flytekit_venv
 
-ENTRYPOINT ["/usr/local/bin/flytekit_venv", "/opt/entrypoint.sh"]
```

Comment:
this one is needed, this is spark, spark needs the /opt/entrypoint.sh
we can document it here